### ISSUE TITLE
SolubilityDescription vs. Solubilty

### ISSUE THREAD

**Issue by Henrich-BGRCI** [esdscom/sdscom-xml#71]
How to describe the unsolubility for a particular medium measured at certain temperature?
E.g., solubility: unsoluble in water at 20 °C
I think, at the moment the only possibility is to write it as Phrase in SolubilityDescription, but it would be more structured, if Solubility (9.2.11.2) would have a field to describe explicitly the unsolubility/low solubility/... (enumeration?). As far as I understand, the field OtherMediumDescription (Phrase) is reserved to describe the kind of medium if it is "Other", isn't it?

**Comment by dirk-qualisys** [comment]
You are right regarding OtherMediumDescription.

I know that unsoluble / nearly unsoluble etc. are quite commonly used, but after consulting with my colleagues, the main question is: How soluble is "low solubility"? "Unsoluble" could be represented as "< 0.01 g/L" if applicable, and any kind of low solubility could be represented in a similar fashion. In addition or alternatively, a human-readable wording finds its place in SolubilityDescription.

The main idea is: **Any values that cannot be represented as a number require human interpretation, so a representation as a phrase is good enough.** Is this approach good enough for you?

**Comment by Henrich-BGRCI** [comment]
I think, if there are no values or user-specified definition given for "low solubility" or "unsoluble", it shouldn't be translated into a numeric value.
In one of the SDS I got, I found another possibility: For the solubility value the datatype PhysChemUnitValueWithTemperature (derived from PhysChemValue) is used, which has the element Comments (Phrase). Here,  "low solubility" or "unsoluble" could be entered as freetext.
